examples/templates/index-dark.html

```diff
@@ -39,7 +49,7 @@
   <div class="osgjs-info osgjs-info-dark">
 
     <a href="http://osgjs.org/" class="osgjs-powered" title="Powered by OSG.JS">
-      <img src="http://osgjs.org/assets/logo.png" alt="" width="32" height="32">
+      <img src="../../website/assets/logo.png" alt="" width="32" height="32">
```

Comment:
arf, indeed. forgot the website. 
Thing is coding without internet on examples gives you a 5 secondes task because of this thing here. perhaps we can add logo to media example folder so that the path is constant ?